06.bowling_object/lib/game.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+class Game
+  attr_reader :frames
+
+  STRIKE_MARK = 'X'
+
+  def initialize(shot_marks)
+    @frames = []
+    shot_pins = shot_marks.split(',').map do |mark|
+      convert_to_i(mark)
+    end
+    build_frames(shot_pins)
```

Comment:
build_framesの中で `@frames` が変更されるのはメソッドの副作用にがっつり依存してしまうのであまり設計としてよくないです。

```suggestion
    @frames = build_frames(shot_pins)
```

のような形にできないか設計を見直してみてください

Reply:
```@frames = build_frames(shot_pins)``` となるように修正しました